addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -333,6 +302,74 @@
                 </div>
             </div>
         </t>
+        <t t-elif="channel.enroll in ['public', 'invite']" name="o_wslides_prerequisite_test">
+            <div t-if="channel.prerequisite_channel_ids and not channel.has_completed_prerequisites"
```

Comment:
Not sure how he could have completed the prerequisites as he is not a member, but maybe I'm missing something.